content/en/docs/tasks/administer-cluster/encrypt-data.md

```diff
@@ -1,52 +1,114 @@
+{{< /note >}}
 
 ## {{% heading "prerequisites" %}}
 
-* {{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
+* {{< include "task-tutorial-prereqs.md" >}}
+
+* This task assumes that you are running the Kubernetes API server as a
+  {{< glossary_tooltip text="static pod" term_id="static-pod" >}} on each control
+  plane node.
 
-* etcd v3.0 or later is required
+* Your cluster's control plane **must** use etcd v3.x (major version 3, any minor version).
 
 * To encrypt a custom resource, your cluster must be running Kubernetes v1.26 or newer.
+  {{< version-check >}}
 
 
 <!-- steps -->
 
-## Configuration and determining whether encryption at rest is already enabled
+## Determine whether encryption at rest is already enabled {#determining-whether-encryption-at-rest-is-already-enabled}
+
+By default, the API server uses an `identity` provider to protect confidential data
+within etcd.
+**The default `identity` provider does not provide any confidentiality protection.**
 
 The `kube-apiserver` process accepts an argument `--encryption-provider-config`
-that controls how API data is encrypted in etcd.
-The configuration is provided as an API named
-[`EncryptionConfiguration`](/docs/reference/config-api/apiserver-encryption.v1/). `--encryption-provider-config-automatic-reload` boolean argument determines if the file set by `--encryption-provider-config` should be automatically reloaded if the disk contents change. This enables key rotation without API server restarts. An example configuration is provided below.
+that specifies a path to a configuration file. The contents of that file, if you specify one,
+control how Kubernetes API data is encrypted in etcd.
+
+The format of that configuration file is YAML, representing a configuration API kind named
+[`EncryptionConfiguration`](/docs/reference/config-api/apiserver-encryption.v1/).
+You can see an example configuration
+in [Encryption at rest configuration](#understanding-the-encryption-at-rest-configuration).
+
+Check whether the `kube-apiserver` process is running with the `--encryption-provider-config`
+command line argument set to `true`. If it is not, you do not have encryption at rest enabled.
+Otherwise, you need to perform further checks to confirm the situation.
+
+If `--encryption-provider-config` is set, check which resources (such as `secrets`) are
+configured for encryption, and what provider is used.
+Make sure that the preferred provider for that resource type is **not** `identity`; you
+only set `identity` (_no encryption_) as default when you want to disable encryption at
+rest.
+Verify that the first-listed provider for a resource is something **other** than `identity`,
+which means that any new information written to resources of that type will be encrypted as
+configured. If you do see `identity` as the first-listed provider for any resource, this
+means that those resources are being written out to etcd without encryption.
```

Comment:
Please help check if the following revision is missing any points we want to make?

```suggestion
If `--encryption-provider-config` is specified, check which resources (such as `secrets`) are
configured for encryption, and which provider is used.
The first provider listed for a resource is the preferred one. If this provider is `identity`,
the resource is **not** encrypted, otherwise any new information written to the resources
of that type is encrypted using the specified provider.
```

Reply:
I think the new text covers this. Would be good to have a tech review confirm that.